document/core/appendix/implementation.rst

```diff
@@ -50,7 +53,8 @@ An implementation may impose restrictions on the following dimensions of a modul
 * the size of a :ref:`structured control instruction <syntax-instr-control>`
 * the number of :ref:`structured control instructions <syntax-instr-control>` in a :ref:`function <syntax-func>`
 * the nesting depth of :ref:`structured control instructions <syntax-instr-control>`
-* the number of :ref:`label indices <syntax-labelidx>` in a |brtable| instruction
+* the number of :ref:`label indices <syntax-labelidx>` in a |BRTABLE| instruction
+* the length of the array in a |ARRAYNEWFIXED| instruction
```

Comment:
AFAICT this is also limited by the bytecode length, but I'm fine specifying it separately this way.

Do we have a limitation on the size of an initializer expression?

Reply:
Good point, added.